From c6058464309a43ecfce43a69f3342a88e513be12 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Mon, 16 Oct 2023 16:59:06 +0200
Subject: [PATCH] Update Makefile

---
 Makefile | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/Makefile b/Makefile
index 9963eff..56708b1 100644
--- a/Makefile
+++ b/Makefile
@@ -157,13 +157,6 @@ all:
 #
 -include $(CURDIR)/make/Makefile
 
-ifneq (0, $(words $(filter mlp libinterface,$(MAKECMDGOALS))))
-# target to build inner blas
-$(LIB_DIR)/lib_mlip_cblas.a:
-	@ $(MAKE) --no-print-directory -f $(MK_THIS) cblas
-
-endif
-
 .PHONY: distclean
 distclean: clean
 	@ $(RM) -f make/config.mk
