From f272b59fabf419788d1936e6e0efd39fbe0b0f61 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Mon, 16 Oct 2023 16:45:45 +0200
Subject: [PATCH] Update configure

---
 configure | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/configure b/configure
index 06e11a2..bfff1b1 100755
--- a/configure
+++ b/configure
@@ -144,9 +144,9 @@ case "${compiler}" in
         fi
         echo "" >> ${MKFILE}
         echo "# Compilers for the library" >> ${MKFILE}
-        echo "CC_LIB  = gcc" >> ${MKFILE}
-        echo "CXX_LIB = g++" >> ${MKFILE}
-        echo "FC_LIB  = gfortran" >> ${MKFILE}
+        echo "CC_LIB  = $(CC)" >> ${MKFILE}
+        echo "CXX_LIB = $(CXX)" >> ${MKFILE}
+        echo "FC_LIB  = $(FC)" >> ${MKFILE}
         ;;
     * ) Halt "unknown compiler \"${compiler}\"." ;;
 esac
@@ -185,7 +185,7 @@ case "${blas}" in
         echo "CXXFLAGS += -DMLIP_INTEL_MKL " >> ${MKFILE}
         ;;
     openblas )
-        echo "LDFLAGS += -L${blasroot}/lib -l:libopenblas.a ${linklib}" >> ${MKFILE}
+        echo "LDFLAGS += -L${blasroot}/lib -lcblas ${linklib}" >> ${MKFILE}
         echo "CPPFLAGS += -I${blasroot}/include" >> ${MKFILE}
         ;;
     embedded )
