From 10a615026c655e3476725f10ab901e9eeee1262a Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Thu, 14 Sep 2023 15:25:28 +0200
Subject: [PATCH] Update configure

---
 configure | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure b/configure
index 7a8758a..237589d 100755
--- a/configure
+++ b/configure
@@ -217,9 +217,9 @@ case "${blas}" in
     openblas )
 	echo "" >> ${MKFILE}
 	echo "# Flags for compilation using OpenBLAS" >> ${MKFILE}
-        echo "LDFLAGS += -L${blasroot}/lib -l:libopenblas.a ${linklib}" >> ${MKFILE}
+        echo "LDFLAGS += -L${blasroot}/lib -lcblas ${linklib}" >> ${MKFILE}
         echo "CPPFLAGS += -I${blasroot}/include" >> ${MKFILE}
-        pythonlibs+=" :libopenblas.a"
+        pythonlibs+=" -lcblas"
         pythonlibdir="${blasroot}/lib"
         pythonincdir="${blasroot}/include"
         ;;
