From a8e3bee5214a628d0169502438036dd35f03737b Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Tue, 9 Feb 2021 10:00:22 +0100
Subject: [PATCH] Update Makefile

---
 make/Makefile | 18 +++++++-----------
 1 file changed, 7 insertions(+), 11 deletions(-)

diff --git a/make/Makefile b/make/Makefile
index 56da73b..80dc3e1 100644
--- a/make/Makefile
+++ b/make/Makefile
@@ -19,9 +19,9 @@ SRCLMP 		= $(SRCPATH)external/MLIP4LAMMPS/lammps_interface.cpp
 HEADERSMLP 	= $(wildcard $(SRCPATH)*.h) $(wildcard $(SRCPATH)common/*.h) $(wildcard $(SRCPATH)drivers/*.h)
 HEADERSUTL 	= $(wildcard $(SRCPATH)utils/*.h)
 
-INCDIR		= ../blas/include
-INCLIB		= ../blas/lib/libopenblas.a
-BLAS_LIBPATH 	= ../blas/lib
+INCDIR		= $(PREFIX)/include
+INCLIB		= $(PREFIX)/lib
+BLAS_LIBPATH 	= $(PREFIX)/lib
 BLAS_SLIB 	= -lopenblas
 BLAS_OBJPATH 	= ../lib/blas_obj/
 
@@ -40,10 +40,10 @@ BIN 		= $(BINPATH)mlp
 
 CC 		= mpicxx
 CC_SRL	= g++
-CFLAGS 		= -O2 -std=c++11 
+# CFLAGS 		= -O2 -std=c++11 
 LINK 		= mpicxx 
 LINK_SRL	= g++
-LINKFLAGS 	= -o 
+LINKFLAGS 	= $(LDFLAGS) -o 
 ARCHIVE 	= ar
 ARCHFLAG1 	= -x
 ARCHFLAG2 	= -rvs
@@ -53,16 +53,12 @@ ARCHFLAG2 	= -rvs
 all:	$(OBJCMN) 
 
 lib: 	$(OBJCMN_SRL) $(OBJLMP)
-	(cd ../blas/OpenBLAS/ && make && make install)
 	mkdir -p ../lib
-	mkdir -p $(BLAS_OBJPATH)
-	cd $(BLAS_OBJPATH) && $(ARCHIVE) $(ARCHFLAG1) ../$(INCLIB) 
-	$(ARCHIVE) $(ARCHFLAG2) $(LIB) $(OBJCMN_SRL) $(OBJLMP) $(BLAS_OBJPATH)*.o 
+	$(ARCHIVE) $(ARCHFLAG2) $(LIB) $(OBJCMN_SRL) $(OBJLMP)
 	rm -f $(OBJCMN_SRL) $(OBJLMP)
 
 mlp:	$(OBJCMN) $(OBJBIN)
-	(cd ../blas/OpenBLAS/ && make && make install)
-	$(LINK) $^  $(LINKFLAGS)  $(BIN) -L$(BLAS_LIBPATH) -lopenblas
+	$(LINK) $^  $(LINKFLAGS)  $(BIN) -L$(BLAS_LIBPATH) -lcblas
 
 mlp_srl:	$(OBJCMN_SRL) $(OBJBIN_SRL)
 	(cd ../blas/OpenBLAS/ && make && make install)
